### ISSUE TITLE
From version-v0.22.106 search movies in Rutracker.Org was broken

### ISSUE THREAD

**Issue by OvchinnikovKirill** [Jackett/Jackett#15470]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
07-07 20:30:45 Info Starting Jackett v0.22.106
07-07 20:30:45 Info Environment version: 8.0.6 (/app/Jackett/)
07-07 20:30:45 Info OS version: Unix 6.5.0.35 (64bit OS) (64bit process)
07-07 20:30:45 Info Jackett variant: CoreLinuxMuslAmdx64
07-07 20:30:45 Info File /etc/issue: Welcome to Alpine Linux 3.20
07-07 20:30:45 Info Running in Docker: Yes (image build: v0.22.106-ls431)
07-07 20:30:45 Info ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
07-07 20:30:45 Info App config/log directory: /config/Jackett
07-07 20:30:45 Info Using proxy: Disabled
07-07 20:30:45 Info Using FlareSolverr: No
07-07 20:30:45 Info Using HTTP Client: HttpWebClient2
```

### Description

From version-v0.22.106 to latest version 0.22.288 search movies in Rutracker.Org was broken.
New version search only in TV Category

### Logged Error Messages

```text
Old version:
https://rutracker.org/forum/tracker.php?nm=dark%25waters
07-07 20:41:10 Info Manual search in RuTracker.org for dark waters => Found 50 releases [2262ms]

New version:
https://rutracker.org/forum/tracker.php?nm=dark%25waters+%D0%A1%D0%B5%D0%B7%D0%BE%D0%BD%3A+
07-07 20:38:55 Info Manual search in RuTracker.org for dark waters => Found 10 releases [1333ms]
```

**Comment by garfield69** [comment]
fixed by #15465

**Comment by garfield69** [comment]
v0.22.301
